doc/examples/pick_place/pick_place_tutorial.rst

```diff
@@ -6,6 +6,8 @@
 Pick and Place
 ==============
 
+NOTE: The functionality used in this tutorial is deprecated. To perform a pick and place operation, MoveIt Task Constructor should be used. Here is a tutorial that goes over how to perform a pick and place operation with MoveIt Task Constructor: `Pick and Place with MoveIt Task Constructor <https://moveit.picknik.ai/humble/doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.html>`_. MoveIt Task Constructor provides a way to plan for tasks that consist of multiple different subtasks, such as a pick and place operation. More information on the MoveIt Task Constructor can be found `here. <https://moveit.picknik.ai/humble/doc/examples/moveit_task_constructor/moveit_task_constructor_tutorial.html>`_
```

Comment:
Bolding would make it stand out a bit more. And it doesn't look like MTC is ever defined on this page, so this is as good a place as any to do it! 
```suggestion
**NOTE:** The functionality used in this tutorial is deprecated. To perform a pick and place operation, MoveIt Task Constructor (MTC) should be used (`Pick and Place with MoveIt Task Constructor <https://moveit.picknik.ai/humble/doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.html>`_). More information on the MoveIt Task Constructor can be found `here. <https://moveit.picknik.ai/humble/doc/examples/moveit_task_constructor/moveit_task_constructor_tutorial.html>`_
```